lib/array.js

```diff
@@ -101,15 +101,32 @@ function selectFromArray (array, key, value) {
   return array.filter(item => value.includes(_.get(item, key)))
 }
 
+/**
+ * Returns only unique items in an array.
+ *
+ * @example
+ * unique(['Orange','Banana','Apple','Orange'])
+ * //Orange,Banana,Apple
```

Comment:
Does this function return an array, or a comma separated string? (I’m guessing it would be an Array)

(Also reminds me that `@returns` comments should include the returned datatype, I’ll go and add those in a separate PR.)

Reply:
All updated